src/components/ToggleArrow/__stories__/ToggleArrow.stories.tsx

```diff
@@ -5,14 +5,70 @@ import ToggleArrow, {ToggleArrowProps} from '../ToggleArrow';
 import data from './data.json';
 
 export default {
-    component: ToggleArrow,
     title: 'Components/ToggleArrow',
-} as Meta;
+    component: ToggleArrow,
+    parameters: {
+        layout: 'centered',
+        controls: {expanded: true},
+    },
+    argTypes: {
```

Comment:
This should work automatically. If it doesn't work, then you need to understand why.

Look at the reasons that I described in the comments, maybe it's them?
https://github.com/gravity-ui/page-constructor/pull/1322